app/models/g5_authenticatable/user.rb

```diff
@@ -44,9 +45,7 @@ def locations
       G5Updatable::LocationPolicy::Scope.new(self, G5Updatable::Location)
                                         .resolve
     end
-
-    private
-
```

Comment:
did you mean to remove the `private` — dont see `add_scoped_role` being called anywhere else

Reply:
Yep, it's getting used in the new class that was created.

Reply:
👍